### ISSUE TITLE
Kiwi automatically closing when no tabs are open

### ISSUE THREAD

**Issue by RaWolFoX** [kiwibrowser/src.next#863]
I disabled "Close browser when no tabs are open". But still browser closes when no tabs are open. Kiwi immediately closes after closing last tab. Its only happened when long tapping on tab switcher button on toolbar and close last tab. Not happening when closing with swipe on tab switching screen.
![Issue.jpg](https://github.com/kiwibrowser/src.next/assets/114433842/7f832ed0-7dcf-4deb-8724-a72437335979)

**Comment by d3ward** [comment]
Is this still an issue ? Are you on latest 111 version ? After disabling "Close browser when no tabs are open" did you restart Kiwi ?

**Comment by RaWolFoX** [comment]
> Is this still an issue ? Are you on latest 111 version ? After disabling "Close browser when no tabs are open" did you restart Kiwi ?
> 

Im on version 112.0.5615.137. Yes this still happens. I think "Close browser when no tabs are open" option is working when closed using tab showing screen(screen that shows tabs as cards). But when long tapping on that tab switching button on status bar and select close will close kiwi entirely. Thats the problem.

**Comment by d3ward** [comment]
Did you tried re-disabling the option ? I just tried on 2 phone and working as expected . If you have Kiwi installed from playstore , can you try Kiwi from github ( download the arm64 with DEV so you will have 2 kiwi installed without the need to uninstall the playstore version )

**Comment by RaWolFoX** [comment]
> Did you tried re-disabling the option ? I just tried on 2 phone and working as expected . If you have Kiwi installed from playstore , can you try Kiwi from github ( download the arm64 with DEV so you will have 2 kiwi installed without the need to uninstall the playstore version )

Ok. I will try.

**Comment by RaWolFoX** [comment]
> Did you tried re-disabling the option ? I just tried on 2 phone and working as expected . If you have Kiwi installed from playstore , can you try Kiwi from github ( download the arm64 with DEV so you will have 2 kiwi installed without the need to uninstall the playstore version )

You are correct. No problem in github version. When will this version come to play store?. Why everything on github is pre release?.

**Comment by RaWolFoX** [comment]
> Currently the latest Github release is the one that got released on play store too 
> Github releases are considered like beta , where user can have more updates , test fixes before actually pushing to play store . 
> At this point I think you need to wait for next play store version that could fix the bug , looks like you got into this bug somehow.

I reinstalled kiwi from play store. now this is not happening. Maybe some setting from previous version caused this issue. Works good now.

**Comment by RaWolFoX** [comment]
> Is this still an issue ? Are you on latest 111 version ? After disabling "Close browser when no tabs are open" did you restart Kiwi ?
> 

Hey i found the culprit. Its the tab switcher. When tab switchr is set to grid or vertical, this bug happens. Can you check this on your device?

**Comment by d3ward** [comment]
That's a nice catch! We will look into it , thank you !
